packages/fmg-core/src/test/framework/transitions.test.ts

```diff
@@ -1,39 +1,51 @@
-import assertRevert from '../helpers/assert-revert';
+import { ethers, ContractFactory, Wallet } from "ethers";
+import linker from "solc/linker";
 
-import { CountingGame } from '../../src/test-game/counting-game';
-import { Channel, State } from '../../src';
+import expectRevert from "../helpers/expect-revert";
 
-const StateLib = artifacts.require("./State.sol");
-const Rules = artifacts.require("./Rules.sol");
-const CountingStateContract = artifacts.require("../test-game/contracts/CountingState.sol");
-const CountingGameContract = artifacts.require("../test-game/contracts/CountingGame.sol");
+import { CountingGame } from "../../test-game/counting-game";
+import { Channel } from "../..";
+
+import StateArtifact from "../../../build/contracts/State.json";
+import RulesArtifact from "../../../build/contracts/Rules.json";
+import CountingStateArtifact from "../../../build/contracts/CountingState.json";
+import CountingGameArtifact from "../../../build/contracts/CountingGame.json";
+
+describe("Rules", () => {
+    const provider = new ethers.providers.JsonRpcProvider("http://localhost:8545");
+    const privateKey = "0x" + "1".repeat(64);
+    const wallet = new Wallet(privateKey, provider);
 
-contract('Rules', (accounts) => {
     let channel;
     let otherChannel;
     let defaults;
     let framework;
     const resolution = [12, 13];
     const otherResolution = [10, 15];
 
+    const participantA = new ethers.Wallet("6cbed15c793ce57650b9877cf6fa156fbef513c4e6134f022a85b1ffdd59b2a1");
+    const participantB = new ethers.Wallet("6370fd033278c143179d81c5526140625662b8daa446c22ee2d73db3707e620c");
+    const participants = [participantA.address, participantB.address];
+
     let fromState;
     let toState;
 
-    before(async () => {
-        framework = await Rules.deployed();
+    beforeEach(async () => {
+        const networkId = (await provider.getNetwork()).chainId;
```

Comment:
It'd be nice to encapsulate this in a helper function that we can share between this repo and rps-poc. Not sure how easy that would be to do due to the linking though.